server/controllers/appointmentController.ts

```diff
@@ -52,16 +53,14 @@ export default class AppointmentController {
   public displayAddAppointment(): RequestHandler {
     return async (req: Request, res: Response, next: NextFunction) => {
       const { clientToken } = req.middleware
-      const {
-        user: { activeCaseLoadId },
-      } = res.locals
       const { prisonerNumber } = req.params
       const { firstName, lastName, bookingId, cellLocation, prisonId } = req.middleware.prisonerData
       const prisonerName = formatName(firstName, undefined, lastName, { style: NameFormatStyle.lastCommaFirst })
+      const user = res.locals.user as PrisonUser
```

Comment:
So this is now forcing us to consider that the user could not be a `PrisonUser` and therefore might not have the concept of a `caseload` or a `staffId`.  Might seem a bit annoying, but it's already flushed out a few things such as the question, 'should non-prison users have access to these pages?'

Here I've cast the type to PrisonUser because the check is done in middleware.  I think that's probably the best way to do it, or we could do a check and `return next`? 🤔